### ISSUE TITLE
Rule Change: prefer-destructuring should have better differentiation between single and multiple destructuring

### ISSUE THREAD

**Issue by arturenault** [eslint/eslint#16370]
### What rule do you want to change?

prefer-destructuring

### What change to do you want to make?

Implement suggestions

### How do you think the change should be implemented?

A new option

### Example code

```js
// Example 1, single destructuring
var foo = object1.foo

// Example 2, multiple destructurings
var bar = object1.bar;
var baz = object1.baz;
```

### What does the rule currently do for this code?

Example 1 gets converted into the following:
```js
var { foo } = object1
```

Example 2 gets converted into the following:
```js
var { bar } = object2
var { baz } = object2
```

### What will the rule do after it's changed?

I personally don't see utility in preferring destructuring for a single property. So I'd like an option to disable that, so that example 1 would be considered correct code.

For example 2, this rule would be much more useful if the code got autofixed to this:
```js
var { bar, baz } = object2
```

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @arturenault, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `prefer-destructuring` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.
